Sources/tart/Commands/Run.swift

```diff
@@ -14,13 +17,35 @@ struct Run: AsyncParsableCommand {
   @Flag var noGraphics: Bool = false
 
   @Flag var recovery: Bool = false
+  
+  @Flag var vnc: Bool = false
```

Comment:
It would be nice to add an explanation on what this flag does, because someone might think that it configures and starts a VNC server automatically for them, which is not true.